spec/system/assessor_interface/completing_assessment_spec.rb

```diff
@@ -410,6 +416,20 @@ def then_the_application_form_is_declined
     )
   end
 
+  def and_i_click_view_timeline
+    assessor_application_page.click_link(
+      "View timeline of this applications actions",
+    )
+  end
+
+  def then_i_see_application_declined
+    expect(
+      assessor_timeline_page.timeline_items.any? do |event|
+        event.title.has_content?("Application declined")
+      end,
+    ).to be_truthy
```

Comment:
This works, but I wonder if this might be a better way of structuring it (to make it clear what the code is doing):

```rb
declined_timeline_event = assessor_timeline_page.timeline_items.find do |timeline_event|
  timeline_event.title == "Application declined"
end

expect(declined_timeline_event).to_not be_nil
```

It's probably a good idea to add a test to the `DeclineQTS` service to check that the timeline event is being created. You can use the [`have_recorded_timeline_event` matcher](https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/main/spec/support/matchers/have_recorded_timeline_event.rb), for example something like this:

https://github.com/DFE-Digital/apply-for-qualified-teacher-status/blob/79708ea2fe9763b7de0955a46c80c1d502a6b57e/spec/services/create_note_spec.rb#L28-L33